common/djangoapps/student/models.py

```diff
@@ -1442,6 +1442,28 @@ def enrollments_for_user(cls, user):
         return cls.objects.filter(user=user, is_active=1)
 
     @classmethod
+    def preloaded_enrollments_for_user(cls, user):
+        """
```

Comment:
Based on chat discussion with @robrap, it's going to be `enrollments_for_user_with_overviews_preload`